cli/internal/benthos/inputs/neosync-connection-data.go

```diff
@@ -231,3 +253,93 @@ func (g *neosyncInput) Close(ctx context.Context) error {
 	g.neosyncConnectApi = nil // idk if this really matters
 	return nil
 }
+
+func convertDynamoDBItemToMap(item map[string]any) (standardMap map[string]any, keyTypeMap map[string]neosyncbenthos_dynamodb.KeyType) {
+	result := make(map[string]any)
+	ktm := make(map[string]neosyncbenthos_dynamodb.KeyType)
+	for key, value := range item {
+		result[key] = convertDynamoDBValue(key, value, ktm)
+	}
+
+	return result, ktm
+}
+
+func convertDynamoDBValue(key string, value any, keyTypeMap map[string]neosyncbenthos_dynamodb.KeyType) any {
```

Comment:
Is this specific to the CLI? Same question as the conn data.
Seems liek we should have the dyanmodb util stuff be centralized in a top level `interanl/dynamodb` module